subsys/metrics/src/main/java/org/commonjava/indy/metrics/zabbix/cache/event/MetricsZabbixCacheEventManager.java

```diff
@@ -0,0 +1,44 @@
+package org.commonjava.indy.metrics.zabbix.cache.event;
+
+import org.commonjava.indy.subsys.datafile.DataFile;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.event.Event;
+import javax.inject.Inject;
+import java.util.concurrent.ConcurrentHashMap;
+
+/**
+ * Created by xiabai on 5/5/17.
+ */
+@ApplicationScoped
+public class MetricsZabbixCacheEventManager
```

Comment:
I'd get rid of this class too, since we can couple the two zabbix cache classes together and avoid the indirection that makes it hard to test.